tests/integration_tests/performance/test_network_tcp_throughput.py

```diff
@@ -33,19 +35,20 @@ def measurements():
 
 def criteria_stats(cpu_baseline: dict, iperf3_id: str, env_id: str):
     """Return statistics definitions based with pass criteria."""
-    baseline_bw = cpu_baseline["baseline_bw"][env_id][iperf3_id]
+    [kernel, rootfs] = env_id.split("/")
```

Comment:
What's the typical format of `env_id`? Could we describe it in a comment?

Reply:
well, I think this line is self-explanatory. Having a comment that says exactly that `env_id` is in this case of format `kernel/rootfs` seems unnecessary.
This `env_id` variable is specific to this test, doesn't have smth to do with the framework.